packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -142,7 +141,6 @@ export default function TemplatePanel() {
 			) }
 			<PostLastRevisionPanel />
 			<PostTaxonomiesPanel />
-			<PostFeaturedImagePanel />
```

Comment:
@youknowriad you have added this here: https://github.com/WordPress/gutenberg/pull/57053/files#diff-6139877d7164dad773ae6521334135a37eb7e0bb351ade4c4f2169aa69da6e34R68, but I don't think it's needed or used anywhere.. 

Am I missing something?